### DISCUSSION TITLE
Sphinx Book Theme 0.4: custom header is missing

### DISCUSSION THREAD

**Discussion by PhilipVinc** [executablebooks/meta#937] [Q&A]
I have a sphinx website that uses sphinx-book-theme 0.3.x to which I added an additional non sticky header written in bootstrap 4.
See [the website](https://netket.readthedocs.io/en/latest/).

The header is taken from an `Hugo` bootstrap website and slightly modified in order to give the illusion that our main website (`https://www.netket.org`) and the `readthedocs` website are the same.

To use this header, I simply added the file [`assets/templates/sections/header.html`](https://github.com/netket/netket/blob/master/docs/assets/templates/sections/header.html) with the relevant HTML code.

However, upon updating to the pre-release of sphinx-book-theme 0.4 this does not work anymore and the extra header is not displayed.

Is there some guide that could help me in the transition?

**Comment by choldgraf** [comment]
I think that this will be fixed when the pydata sphinx theme makes their next release, but the short version is: double check that the header doesn't have `display: none` attached to it. For now, I think that rule is active and that might cause your problem.

This PR is doing the updates for the new pydata theme if you wanna iterate over there:

- https://github.com/executablebooks/sphinx-book-theme/pull/677
